internal/hmyapi/blockchain.go

```diff
@@ -225,3 +225,9 @@ func doCall(ctx context.Context, b Backend, args CallArgs, blockNr rpc.BlockNumb
 	}
 	return res, gas, failed, err
 }
+
+// LatestHeader returns the latest header information
+func (s *PublicBlockChainAPI) LatestHeader(ctx context.Context) *HeaderInformation {
+	header, _ := s.b.HeaderByNumber(context.Background(), rpc.LatestBlockNumber) // latest header should always be available
```

Comment:
what is this rpc.LatestBlockNumber? why don't we just get the current block and return the header? as the func is "LatestHeader"

Reply:
since we have a rpc call to get latest block number:  "blockNumber", it used this function to get latest blocknumber, this is where I borrow the code.